doc/nrf-bm/app_dev/dfu/memory_partitioning.rst

```diff
@@ -0,0 +1,74 @@
+.. _dfu_memory_partitioning:
+
+Memory Partitioning for DFU
+###########################
+
+In order to successfully run the DFU process, the memory of the device must be first correctly partitioned to accommodate all required components.
+
+|BMshort| uses Zephyr's `DTS`_ system for memory partitioning.
+You can find a practical example of how to partition the memory in chapter :ref:`ug_dfu_preparing_dfu_board`.
+
+Partitions overview
+*******************
+
+The following are detailed descriptions of each memory partition needed when running Firmware Over The Air (FOTA) updates with MCUboot as the Initial Root of Trust (IRoT).
+
+.. figure:: ../../images/partition_RRAM.png
+   :alt: Memory partitioning with FOTA and mcuBoot as IRoT
+
+The memory in the device is divided into several partitions, each serving a specific function in the DFU process:
+
+.. list-table::
+   :header-rows: 1
+
+   * - Partition name
+     - Purpose
+     - Details
+   * - boot_partition
+     - Reserved exclusively for IRoT.
+     - Starts at address 0x0000, size is determined by the needs of the mcuBoot firmware.
+   * - storage_partition (optional)
+     - Used for application data storage that persists across resets and firmware updates.
+     - Placement and size are configurable based on application requirements.
+   * - slot0_partition
+     - Contains the main application firmware.
+     - Begins immediately after the storage partition or the IRoT partition if the storage is not enabled. The size is maximized up to the start of the firmware loader.
+   * - slot1_partition
+     - Serves as the location for the DFU mode and contains the firmware loader and the SoftDevice.
+     - This partition is split into two to independently validate the images of the firmware loader and the SoftDevice, which is shared with the application.
```

Comment:
Bit hard to read. Suggest rewriting:
""This partition is split into two independent sub-partitions FW Loader and SoftDevice that will be validated independently by IRoT. The FW Loader is started when entering DFU mode, while SoftDevice is used by both FW Loader as well as the Application running from slot_0."